src/main/java/com/kodilla/ecommercee/dao/UserDao.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.ecommercee.dao;
+
+import com.kodilla.ecommercee.User;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+
+@Transactional
+@Repository
+public interface UserDao extends CrudRepository<User, Long> {
```

Comment:
powinno być "UserRepository" zamiast "UserDao"

Reply:
w sumie warstwy dostępu też nie ustaliliśmy, czy idziemy w dao, czy w repository...
zapewne będzie z rana do obgadania ;)

Reply:
no to poszedłem za tłumem i zmienione na Repository :)